test/freenet/crypt/CryptByteBufferTest.java

```diff
@@ -259,14 +264,14 @@ public void testSuccessfulRoundTripByteArrayReset() throws GeneralSecurityExcept
                 ByteBuffer decipheredtext1 = crypt.decryptCopy(ciphertext1);
                 ByteBuffer decipheredtext2 = crypt.decryptCopy(ciphertext2);
                 ByteBuffer decipheredtext3 = crypt.decryptCopy(ciphertext3);
-                assertTrue("CryptByteBufferType: "+type.name(), plain.equals(decipheredtext1));
-                assertTrue("CryptByteBufferType: "+type.name(), plain.equals(decipheredtext2));
-                assertTrue("CryptByteBufferType: "+type.name(), plain.equals(decipheredtext3));
+                assertThat("CryptByteBufferType: "+type.name(), plain, equalTo(decipheredtext1));
+                assertThat("CryptByteBufferType: "+type.name(), plain, equalTo(decipheredtext2));
+                assertThat("CryptByteBufferType: "+type.name(), plain, equalTo(decipheredtext3));
             }
     }
     
     private void assertNotEquals(Object o1, Object o2) {
```

Comment:
Please inline this method, it's only used in a handful of places and from its signature it's unclear how it handles arrays.

Reply:
Inline which method? 😕

Reply:
@bertm do you mean assertNotEquals?